### DISCUSSION TITLE
Getting user teams has different object model

### DISCUSSION THREAD

**Discussion by Nevinia210** [microsoftgraph/msgraph-sdk-dotnet#1925] [Q&A] [answered]
Hello,
I have a problem with recent updates in our solution regarding getting teams from `GraphServiceClient`
Previously we had code:
```C#
var teams = (await _graphServiceClient
                    .Users[user.Id]
                    .JoinedTeams.Request().GetAsync())
                    .CurrentPage;

return teams.Select(team => 
                    new GraphTeamDTO
                    {
                        Id = team.Id, 
                        Name = team.AdditionalData["displayName"].ToString(), 
                        Description = team.AdditionalData["description"].ToString()
                    }
                ).ToList();
```
Selected Properties "displayName" and "description" were in the Teams model under AdditionalData.
Currently with the same code, after update, properties are missing. They are no longer key-value pair in AdditionalData, instead, they are visible as normal properties of runtime team object

![image](https://github.com/microsoftgraph/msgraph-sdk-dotnet/assets/42325625/1609f9d0-2c88-46e9-bded-e8a0348c7dc8)

but without debugging mode, properties are not in the model

![image](https://github.com/microsoftgraph/msgraph-sdk-dotnet/assets/42325625/091c8a8e-220f-4bc3-973c-36ff4a6aaa67)

The only change that i did was adding PnP.Framework package (+ some functionality coded in there)

Microsoft.Graph version used: 3.8.0
New PnP.Framework version added: 1.12.0

Can someone please advise me, what am i missing? how to get "displayName" and "description" properties?

**Comment by andrueastman** [comment, marked as answer]
It looks like the `PnP.Framework` depends on [Microsoft.Graph](https://www.nuget.org/packages/Microsoft.Graph/) (>= 3.33.0) while your project depends on 3.8.0.
![image](https://github.com/microsoftgraph/msgraph-sdk-dotnet/assets/6464005/d4aef4e0-d79b-40bc-a0c4-ec9231f5cb8a)

This is causing the project to use the higher package version referred to by `PnP.Framework` by which defines the properties and are therefore not being placed in the additionaldata. 

I would suggest updating you project reference to 3.33.0 so that the reference match and you can simply read the properties from the Team object consisitently.

**Reply by Nevinia210** [reply]
yep, that did the trick, thanks :)
